utils/ifupdown.sh.in

```diff
@@ -46,6 +46,7 @@ elif [ "$MODE" = 'stop' ]; then
   "$ip" link set dev "$IFACE" down || exit 2
 fi
 
+# shellcheck disable=SC2086
```

Comment:
You removed the all_interfaces= && line in a later commit.  Does this disable line specifically apply to the all_interfaces= line, or does it apply to other lines as well?